lib/fhir/query-utils.js

```diff
@@ -237,7 +244,119 @@ module.exports = (mongo) => {
       }
       return addToClause(value)
     },
+*/
+
+    nameToMongoClause: (fieldToMatch, value, modifier) => {
```

Comment:
We normally add test for each new resource we add and all search parameters for this. But in this case that is a lot of work. So, I'd suggest only writing tests for the search parameters were we use these new query utils functions so that we get coverage of these functions and ensure they do what we expect.